docs/developer-guide.MD

```diff
@@ -26,3 +26,9 @@ configurators:
     - {"task_name": {"res": False, "err": str(ex)}} - on failure
 - Add the new module to [configurators_mappings.py](../openshift_day2_configurator/configurators/mappings/configurators_mappings.py)
   under `configurators_mappings` dictionary.
+
+# Tests
+#### Run day2 configurator tool tests:
+```bash
+poetry run pytest openshift_day2_configurator/tests
+```
```

Comment:
please add that new code should be covered by tests

Reply:
done